base/applications/sndvol32/dialog.c

```diff
@@ -373,7 +373,7 @@ LoadDialog(
     Offset++;
 
     /* calculate font length */
-    Length = wcslen((LPWSTR)Offset) + 1;
+    Length = (WORD)wcslen((LPWSTR)Offset) + 1;
```

Comment:
Why don't you declare Length as size_t instead ?

Reply:
Good question. Will fix.